assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu22.04:latest
+
+COPY requirements.txt /app/requirements.txt
+RUN pip install -r /app/requirements.txt
+
+# Copy your Python file into the image
+COPY evaluate.py /app/evaluate.py
```

Comment:
in the service, the command is
`python /app/evaluate_on_data.py`
Could we name the python file on the container the same so we don't have to do a release cycle in the service to consume this registry environment?

Reply:
Ah, this was renamed to evaluate because of feedback from @janhavi13. Just to confirm, should we name this "evaluate_on_data" for now and then rename later in the next version?